src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -218,6 +329,12 @@ private static ConsoleColor ColorAttributeToConsoleColor(Interop.mincore.Color c
             return (ConsoleColor)c;
         }
 
+        private static Interop.mincore.CONSOLE_SCREEN_BUFFER_INFO GetBufferInfo()
+        {
+            bool unused;
+            return GetBufferInfo(true, out unused);
+        }
+
```

Comment:
Are there any tests we can write?  If nothing else the tests could at least execute the code paths to ensure they don't throw when they're not supposed to.